dotCMS/src/integration-test/java/com/dotcms/enterprise/rules/RulesAPIImplIntegrationTest.java

```diff
@@ -82,12 +79,80 @@ public void shouldSaveCondition() throws DotSecurityException, DotDataException
         final Rule rule = new RuleDataGen().host(host).nextPersisted();
 
         final ConditionGroup conditionGroup = new ConditionGroupDataGen().rule(rule).next();
-        this.addPermission(role, host, true);
+        this.addRulesPermission(role, host, true);
```

Comment:
create a variable for the third parameter to understand what's going on
`boolean addPublishPermsForRules = true;`

Reply:
Or just seperate `addRulesPermission` into two methods with clearer responsabilities:
`addPermsToPublishRules`
`addPermsToReadRules`

Reply:
done https://github.com/dotCMS/core/pull/18201/commits/a3d5384f514af695b89fdbc549338def9a8cce2c#diff-9baded4fb0fb63e2648d9c9c4380ec1aR889-R910